index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-container">
+        <div class="time-network">
+          <h4>9:41</h4>
+          <ul class="network-only">
+            <li class="network-only-item">
+              <img src="image/CellularConnection.png" alt="" />
+            </li>
+            <li class="network-only-item">
+              <img src="image/Wifi.svg" alt="" />
+            </li>
+            <li class="network-only-item">
+              <img src="image/Capacity.svg" alt="" />
+            </li>
+          </ul>
+        </div>
```

Comment:
- [ ] [Optional] Kindly remove this div as it is not professional for your portfolio to have this div.
![Screenshot from 2022-04-21 22-26-16](https://user-images.githubusercontent.com/33752735/164547657-340651ff-4071-4683-82ea-38e5ccf95bcd.png)